src/components/interface/SectionHeader.jsx

```diff
@@ -12,7 +12,8 @@ const SectionHeader = ({
   classes,
   title,
   showNew,
-  showWinner
+  showWinner,
```

Comment:
we don't need the showWinner component anymore so we can remove any logic that has to do with showWinner